libraries/SITL/SIM_Sailboat.cpp

```diff
@@ -184,7 +184,7 @@ void Sailboat::update(const struct sitl_input &input)
     // calculate apparent wind in earth-frame (this is the direction the wind is coming from)
     // Note than the SITL wind direction is defined as the direction the wind is travelling to
```

Comment:
We may be able to remove this note now. I suspect if it needed changing here we might also have to look at the SITL windvane backend.

Reply:
can you check the SITL windvane code? I'm not familiar with it